utils/video.py

```diff
@@ -53,3 +56,17 @@ def add_watermark(self, text, redditid, opacity=0.5, duration: int | float = 5,
         # Overlay the img clip on the first video clip
         self.video = CompositeVideoClip([self.video, img_clip])
         return self.video
+
+    def add_overlay(self):
+            # Get duration for the entire video to place the overlay at the correct time
+            video_duration = self.video.duration
+
+            overlayName = settings.config["settings"]["sub_overlay_name"]
```

Comment:
Move that to a kwarg or arg

Reply:
Thanks for reviewing this PR! I'm a beginner at Python and I don't see the benefit of doing that. Could you please explain?

Reply:
It follows the code design better

So instead of accessing the config in the function
Pass it in via a keyword argument (kwarg)